src/includes/load-service.h

```diff
@@ -150,18 +150,100 @@ inline string_iterator skipws(string_iterator i, string_iterator end) noexcept
     return i;
 }
 
-// Convert a signal name to the corresponding signal number
-inline int signal_name_to_number(std::string &signame) noexcept
+// Utilities to map strings/signals to int
+// The signal list is created statically with only those signals,
+// which are defined for the current architecture/system
+const std::pair<const char *, int> signal_to_int_map[] = {
```

Comment:
Signal list has not been trimmed yet, why did you mark as ready for review?

Reply:
Please don't request review until it is actually ready. If you have questions in the meantime you can ask them.

Reply:
Sorry, I mean "Hey everything looks good and let's talk about signal list" but my opinion was changed; regardless than that, everything seems ok in my opinion.

Reply:
Until it's at the stage where you think that no more changes are needed, I'd prefer not to be reviewing it. Reviewing takes up a lot time (I know that's probably not obvious, but I have to be very careful and check a lot of things). So, please only request review when you think it's finished, and after you've carefully reviewed it yourself. If you're not totally sure that it's ok that's fine, but if there are definitely going to be changes needed then don't ask for review, instead ask questions to clarify those points.

Reply:
(So to be clear, if we need to talk about the signal list, that means it's not ready for review.)